helpers.sh

```diff
@@ -47,21 +47,6 @@ llvm_latest_version() {
   echo "17"
 }
 
-# $1 - toolchain name
```

Comment:
given that you have provided the default everywhere, this may be safe to remove directly. BUT I think you would need to  change the action `get-llvm-version` to return that default value because we will not be able to land libbpf/ci changes and vmtest changes (which consume those actions) at the same time.

Reply:
Can we break it for a few minutes...?

Reply:
Actually, `get-llvm-version` still works in current version. So I think we are good? We do change the meaning of `toolchain` though.

Reply:
correct. It is calling `llvm_default_version`, which is still here.

> We do change the meaning of toolchain though.

I think this is fine. We don't do a great job at naming things currently, and should provably fix this. I think it makes sense that `toolchain` becomes `llvm|gcc` and `llvm_version` the actual version.